advisories/github-reviewed/2024/02/GHSA-2cjh-75gp-34gc/GHSA-2cjh-75gp-34gc.json

```
@@ -7,7 +7,7 @@
     "CVE-2024-22859"
   ],
   "summary": "livewire Cross-Site Request Forgery vulnerability",
-  "details": "Cross-Site Request Forgery (CSRF) vulnerability in livewire before v3.0.4, allows remote attackers to execute arbitrary code getCsrfToken function.",
+  "details": "> Cross-Site Request Forgery (CSRF) vulnerability in livewire before v3.0.4, allows remote attackers to execute arbitrary code getCsrfToken function.\n\nReading the advisories this only affects livewire v3 and not v2. This advisory is hence too broad as it also includes v2.\nSuggesting to reduce the scope to only livewire v3",
```

Didn't mean to update the description. 

```suggestion
  "details": "Cross-Site Request Forgery (CSRF) vulnerability in livewire before v3.0.4, allows remote attackers to execute arbitrary code getCsrfToken function.",
```